rosidl_generator_py/cmake/find_valid_typesupport.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
Aesthetically, which do you think is better, prepending the macro with the package name (the macro name) or not (the file name)?